packages/valory/skills/decision_maker_abci/behaviours/sampling.py

```diff
@@ -48,16 +47,7 @@ def __init__(self, **kwargs: Any) -> None:
 
     def setup(self) -> None:
```

Comment:
@Adamantios  I have emptied setup as randomness is no longer a factor in deciding upon a bet